presidio-image-redactor/Pipfile.lock

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
Why the re-lock? It seems like different packages are installed but the Pipfile hasn't changed.

Reply:
re-run lock on wsl to get this change and resolve the build issues in python 3.6 and 3.7